policy-service/README.md

```diff
@@ -13,17 +13,164 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 --->
+# <img src="../logos/logo.svg" width="180">
+
+
 # Policy Service
 
-<span style="color:red">**Note:** As noted in the [documentation root](../README.md) Palisade is in an early stage of development, therefore the precise APIs contained in these service modules are likely to adapt as the project matures.</span>
+The responsibilities of the Policy Service is to provide the set of rules (filters or transformations) that need to be
+applied to each resource that has been requested, based the User and Context.
+
+## High Level Architecture
+
+<!--- 
+See policy-service/doc/policy-service.drawio for the source of this diagram
+--->
+![Policy Service diagram](doc/policy-service.png)
+
+## Message Model
+
+| PolicyRequest           | PolicyResponse           | AuditErrorMessage |
+|:------------------------|:-------------------------|:------------------|
+| *token                  | *token                   | *token            |
+| userId                  | userId                   | userId            |
+| resourceId              | resourceId               | resourceId        |
+| context                 | context                  | context           |
+| user                    | user                     | exception         |
+| resource                | resource                 | serverMetadata    |
+|                         | rule                     |                   |
+
+(fields marked with * are acquired from headers metadata)
+
+The service accepts a `PolicyRequest` and a token, then checks for rules, or policies in the cache; the technology of which is chosen in the Policy Services application.yaml
+and Spring profiles. First, the Policy Service looks for resource level rules, these are rules that can be applied to any part of the resource path, and could limit what resources 
+the User has access to. The Policy Service recursively checks the parents of the resource for any record level rules, and applies them to the resource before continuing.
```

Comment:
Understood that the convention with mf files was to have each sentence on one line.  This looks like the start of a new line in mid-sentence.

Reply:
Theres no real convention. Each browser will format the Readme appropriatley and wrap lines.